lib/babe/median_test.go

```diff
@@ -207,25 +191,18 @@ func TestEstimateCurrentSlot(t *testing.T) {
 }
 
 func TestGetCurrentSlot(t *testing.T) {
-	babesession, dbSrv := createTestSessionWithState(t, nil)
-
-	defer func() {
-		err := dbSrv.Stop()
-		if err != nil {
-			t.Fatal(err)
-		}
-	}()
+	babesession := createTestSession(t, nil)
 
 	// 100 blocks / 1000 ms/s
 	// TODO: use time.Duration
-	addBlocksToState(t, babesession, 100, dbSrv.Block, uint64(time.Now().Unix())-(babesession.config.SlotDuration/10))
+	addBlocksToState(t, babesession, 100, babesession.blockState, uint64(time.Now().Unix())-(babesession.config.SlotDuration/10))
 
 	res, err := babesession.getCurrentSlot()
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	expected := uint64(101)
+	expected := uint64(100)
```

Comment:
so the memory database resolved this as well? nice!

Reply:
yeah, it sped up the gets from the DB